src/file.py

```diff
@@ -0,0 +1,36 @@
+import os
+import shutil
+
+# Copy the file from the specified source path to destination path, the return Boolean value is indicating if the operation succeeded.
+def copy_file(sourcePath: str, destinationPath: str) -> bool:
+    try:
+        shutil.copy(sourcePath, destinationPath)
+        return True
+    except:
+        return False
+
+# Copy the file/directory tree from the specified source path to destination path, the return Boolean value is indicating if the operation succeeded.
+def copy_file_tree(sourcePath: str, destinationPath: str) -> bool:
+    try:
+        shutil.copytree(sourcePath, destinationPath)
+        return True
+    except:
+        return False
+
+# Remove the file at the specified target path, the return Boolean value is indicating if the operation succeeded.
+def remove_file_tree(targetPath: str) -> bool:
+    try:
+        shutil.rmtree(targetPath)
+        return True
+    except:
+        return False
+
+# Create a new directory at the specified target path, the return Boolean value is indicating if the operation succeeded.
+def create_directory(targetPath: str) -> bool:
```

Comment:
This has been implemented in order to maintain consistency in the use of the result-pattern in the builder class, so that any exceptions from the builder are "manually" thrown. It also gives you the opportunity to implement a different approach to exceptions related to file system operations depending on the operating system used.